ops/deployment/docker-compose.yml

```diff
@@ -148,13 +148,27 @@ services:
     extra_hosts:
       - "gigadb.dev:172.16.238.10"
 
+  chrome:
```

Comment:
Need to add chrome to the depends_on section in webapp service.

Reply:
`chrome` is added to `webapp` service, and the Behat tests pass after the change. Should I remove `PhantomJS` from the `webapp`? And  running a Behat test, for example, `1 scenario 9 steps` takes `4m14.62s`,  it is too time consuming?

Reply:
I think we should remove PhantomJS since it is not being maintained anymore and it has been replaced with Chrome in this PR for running Behat tests involving Javascript. Perhaps @rija can confirm we can do this?